### PR TITLE
Merging relnotes for v0.9.1909.3002

### PR DESCRIPTION
**Pull request by bromeg**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/27d37478207374264d3717f28fa71926f74f64be/articles/relnotes/index.md) | 75 | 52 | 156 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f246bb50-6c70-476d-a7d7-d2ef39eb36f9_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/da727a120c42debf482733ea7956a51449ce9b41/articles/relnotes/index.md) | 75 | 52 | 156 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4a89e03e-3a13-4cec-9ea6-816fa317ef2e_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/relnotes/index.md [hunk 1]
```diff
@@ -25,7 +25,20 @@ dotnet tool update -g Microsoft.Quantum.IQSharp
...
-# Version 0.9 (*PackageReference 0.9.1908.2906*)
+# Version 0.10 (*PackageReference 0.10.XXX.YYY*)
+
+*Release date: September 26th, 2019*
+
+This release contains the following:
+
+- New support for Q# code completion in Visual Studio 2019 (versions 16.3 & later) & Visual Studio Code
+- New support for installing the QDK for Python using Conda packages, new installation instructions can be found [here](xref:microsoft.quantum.install) 
+- New [Quantum Kata](https://github.com/Microsoft/QuantumKatas) for quantum adders
```
Comment (tcNickolas) [inline articles/relnotes/index.md]: I would not include new kata in the release notes - technically the new katas were added while still on 0.9. In the blog post it makes sense to announce them, but that's more of a "new and interesting" section than actual part of the release.

File: articles/relnotes/index.md [hunk 2]
```diff
@@ -25,7 +25,20 @@ dotnet tool update -g Microsoft.Quantum.IQSharp
...
+
+See the full list of closed PRs for [libraries](https://github.com/Microsoft/QuantumLibraries/pulls?q=is%3Apr+is%3Aclosed), [compiler](https://github.com/microsoft/qsharp-compiler/pulls?q=is%3Apr+is%3Aclosed), [runtime](https://github.com/microsoft/qsharp-runtime/pulls?q=is%3Apr+is%3Aclosed), [samples](https://github.com/Microsoft/Quantum/pulls?q=is%3Apr+is%3Aclosed) and [Katas](https://github.com/microsoft/QuantumKatas/pulls?q=is%3Apr+is%3Aclosed).  
+
+
+# Version 0.9 (*PackageReference 0.9.1908.2902*)
```
Comment (tcNickolas) [inline articles/relnotes/index.md]: I think the version we had earlier was correct?
```suggestion
# Version 0.9 (*PackageReference 0.9.1908.2906*)
```

File: articles/relnotes/index.md [hunk 3]
```diff
@@ -25,7 +25,20 @@ dotnet tool update -g Microsoft.Quantum.IQSharp
 > * In order to work with a solution in Visual Studio, all projects contained in the solution need to be in the same folder as the solution or in one of its subfolders.  
 > * References between projects migrated to 0.6 and higher and projects using older package versions are **not** supported.
 
-# Version 0.9 (*PackageReference 0.9.1908.2906*)
+# Version 0.10 (*PackageReference 0.10.XXX.YYY*)
```
Comment (cgranade) [inline articles/relnotes/index.md]:
```suggestion
# Version 0.10 (*PackageReference 0.9.1909.3002*)
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cfbee9404e63fa924f5e6361c7b70c4717840531/articles/install-guide/index.md) | 96 | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a8794822-026e-43a6-a0ba-e943eba4659c_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cfbee9404e63fa924f5e6361c7b70c4717840531/articles/relnotes/index.md) | 75 | 50 | 157 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85bb35ef-ab45-43c8-b48c-5e76717a4983_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1ef7ea64f3535568cbf42cd2fc1c5863a851488/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9ca89a6c-7b83-46d4-8e28-584628be5c6b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1ef7ea64f3535568cbf42cd2fc1c5863a851488/articles/install-guide/index.md) | 96 | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7f790987-aade-4c9e-9b4e-8d55ba6661b3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1ef7ea64f3535568cbf42cd2fc1c5863a851488/articles/relnotes/index.md) | 75 | 50 | 157 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/66f083de-016c-46e0-8ed4-a4d443f9da44_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/install-guide/index.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: Learn how to install the Microsoft Quantum Development Kit (QDK)
 author: natke
 ms.author: nakersha
-ms.date: 9/23/2019
+ms.date: 9//2019
```
Comment (cgranade) [inline articles/install-guide/index.md]:
```suggestion
ms.date: 9/30/2019
```

File: articles/install-guide/index.md [hunk 5]
```diff
@@ -182,6 +185,9 @@ Depending on your chosen development environment, there are different installati
         - Go to **Debug** -> **Start Without Debugging**
         - You should see the following text in the output window `Hello quantum world!`
 
+> [!NOTE]
+> * If you have multiple projects within one VS Code workspace, all projects contained in the workspace need to be in the same folder as the solution, or in one of its subfolders.
```
Comment (cgranade) [inline articles/install-guide/index.md]:
```suggestion
> * Workspaces with multiple root folders are not currently supported by the Visual Studio Code extension. If you have multiple projects within one VS Code workspace, all projects need to be contained within the same root folder.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e9dcfe4631c8a15a72a3318140325e67dffa7be9/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bb15c6cf-122a-4852-b2f9-89c7cf83d524_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e9dcfe4631c8a15a72a3318140325e67dffa7be9/articles/install-guide/index.md) | 95 | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/99cb0a80-4c50-4045-87ee-2247bf297d40_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e9dcfe4631c8a15a72a3318140325e67dffa7be9/articles/relnotes/index.md) | 75 | 50 | 157 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d47f38c6-79a8-4e83-b2bc-2be6f03f2622_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9ef1543d4f00e0cc10c697583315aa082283c4b3/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c47e061b-c243-4fc0-825b-74c916d9705e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9ef1543d4f00e0cc10c697583315aa082283c4b3/articles/install-guide/index.md) | 96 | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd27afb1-4856-46d4-b896-0004e482e06a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9ef1543d4f00e0cc10c697583315aa082283c4b3/articles/relnotes/index.md) | 75 | 50 | 157 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e841c5bf-2a25-4066-8f5b-a5fbe2b071c3_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Pull request merged**